### PR TITLE
fix helm wording

### PR DESCRIPTION
**Pull request by letzya**
fix the comment to add helm repo and not clone it
improve the formatting of code

- [x] Merge to latest

### CODE REVIEW

**Review by carolwkc — DISMISSED** [review]
Could you also fix line 60-68? It doesn't has correct line breaks in preview mode.

**Comment by carolwkc** [comment]
@Keithwachira can you review if the updates would help installation from your recent experience?

**Review by yurisasuke — APPROVED** [review]

File: tyk-docs/content/tyk-cloud/environments-&-deployments/hybrid-gateways.md [hunk 1]
```diff
@@ -42,40 +40,35 @@ please use [GitHub Tyk-helm-chart repo](https://github.com/TykTechnologies/tyk-h
...
 
-2. Before we proceed with installation of the chart we need to set some custom values. To see what options are configurable on a chart and save that options to a custom values.yaml file run:
+2. Before proceeding with installation of the chart we need to set some custom values. First save the full original values.yaml to a local copy:
 
 ```bash
 helm show values tyk-helm/tyk-hybrid > values.yaml
 ```
 
-3. For Tyk-hybrid chart we need to modify following values in your custom values.yaml file:
+3. For tyk-hybrid chart we need to modify following values in your custom values.yaml file: `gateway.rpc.rpcKey`, `gateway.rpc.apiKey`, `gateway.rpc.connString`.
```
Comment (letzya) [inline tyk-docs/content/tyk-cloud/environments-&-deployments/hybrid-gateways.md]:
```suggestion
3. For tyk-hybrid chart we need to modify the following values in your custom `values.yaml` file: 
- `gateway.rpc.rpcKey`, 
- `gateway.rpc.apiKey`, 
- `gateway.rpc.connString`.
```

**Review by yurisasuke — APPROVED** [review]

**Pull request merged**